components/email_aliases/resources/content/email_aliases_signin_page.tsx

```diff
@@ -69,7 +79,8 @@ const BeforeSendingEmailForm = ({
       </div>
       <LoginRow>
         <StretchyInput
-          autofocus
+          id='email-input'
+          ref={emailRef}
```

Comment:
nit: this should be a callback ref as we only need it once:


```suggestion
          ref={(el: HTMLInputElement | null) => {
            el?.focus()
          }}
```